packages/block-editor/src/components/grid/style.scss

```diff
@@ -173,18 +172,53 @@
 	}
 }
 
+.editor-collapsible-block-toolbar {
+	.block-editor-grid-item-mover__move-vertical-button-container {
+		// Move up a little to prevent the toolbar shift when focus is on the vertical movers.
+		@include break-small() {
+			height: $grid-unit-50;
+			position: relative;
+			top: -5px; // Should be -4px, but that causes scrolling when focus lands on the movers, in a 60px header.
+		}
+	}
+}
```

Comment:
Similar to [this approach](https://github.com/WordPress/gutenberg/blob/159d01a01b61ee76159ab67bc94d0d7cb3d90b35/packages/editor/src/components/collapsible-block-toolbar/style.scss#L65-L72) for the default block mover. the `.editor-collapsible-block-toolbar` selector is used to determine that the Top Toolbar is enabled.